chord_metadata_service/experiments/migrations/0009_v4_1_0.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Tuple
+from django.db import migrations
+
+LIB_STRATEGY_CONVERSIONS: List[Tuple[str, str]] = [
+    # Convert WES -> WXS ...
+    ("WES", "WXS"),
+    ("Other", "OTHER"),
+]
+
+LIB_SELECTION_CONVERIONS: List[Tuple[str, str]] = [
+    ("Random", "RANDOM"),
+    ("Random PCR", "RANDOM PCR"),
+    ("Exome capture", "other"), # 'Exome capture' no longer supported
```

Comment:
this should be hybrid selection / hybrid capture or something like that, not other - see https://gatk.broadinstitute.org/hc/en-us/articles/360035890911-Hybrid-selection-exome-preparation- or this random WES experiment: https://www.ncbi.nlm.nih.gov/sra/SRX21780723[accn]

but we may want to keep exome capture as a value since its more specific, im not sure

Reply:
Changed to "Hybrid Selection"